spec/services/open_graph_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+describe OpenGraph, type: :service, vcr: true do
+  VCR.use_cassette("open_graph") do
+    let(:page) { described_class.new("https://github.com/forem") }
+  end
+
+  describe "meta-programmed methods" do
+    it "calls the methods" do
+      expect(page.title).to eq("Forem · GitHub")
+      expect(page.author).to be_nil
+      expect(page.description).to include("Forem has 18 repositories available. Follow their code on GitHub.")
+    end
+  end
+
+  describe ".meta_for" do
+    it "gets a specific meta value" do
+      expect(page.meta_for("twitter:card")).to eq("summary_large_image")
+      expect(page.meta_for("enabled-features")).to eq("MARKETPLACE_PENDING_INSTALLATIONS")
+      expect(page.meta_for("theme-color")).to eq("#1e2327")
+    end
+  end
+
+  describe "twitter" do
+    it "returns twitter data" do
+      expect(page.twitter["twitter:site"]).to eq "@github"
```

Comment:
My heart kind of wants `page.twitter_card_site`; but only kind of.

The guiding principle of the OpenGraph is to minimize the implementation of the meta inspector gem's details that our application knows.

Reply:
What you are recommending here is unclear.
Do you mean that I should create methods for each of the Twitter meta_data keys?
For example, `twitter_card_site` for the key `page.twitter["twitter:site"]`?